### ISSUE TITLE
Repeated press of Cmd+Enter in Mac app does not trigger submit

### ISSUE THREAD

**Issue by stefannilsson** [a85/POSTMan-Chrome-Extension#3192]
## App Details:
```
Postman for Mac
Version 5.0.0 (5.0.0)
OS X 10.12.5
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:
**Always**
 2. Expected behaviour:
**Repeated submit of form when pressing Cmd+Enter**

Description:
When submitting a form by pressing *Cmd+Enter*, it's only possible to do so *once* from the Mac app .
Seems some focus is changed to some view hence a second press of *Cmd+Enter* does not end up the request send again. Which is the expected.

**Comment by SamvelRaja** [comment]
This seems to be working from my end. We shall investigate further and let you know the progress here.

**Comment by sdnts** [comment]
Hi @stefannilsson Is there a way to reproduce this? That might help us fix this ASAP :)

**Comment by stefannilsson** [comment]
I apologise for the unclarity of the reproduction procedure. I now noticed it's totally possible to do a repetitive submit by pressing *Cmd+Enter*, if the marker/focus was last in the URL-field.

My scenario:
* Open a new request (i.e. from Collections).
* Press *Send* and wait for the response.
* Select some text from a text-based response (in my case, I copied some data)
* *Submit* by pressing *Cmd+Enter* (Works, the first time!)
* Try submit request again via *Cmd+Enter* (Doesn't work!)

Thanks for a fast feedback!
